.ci/basic_python_requirements

```diff
@@ -3,3 +3,4 @@ alembic==0.9.2
 thrift==0.9.1
 portalocker==1.1.0
 psutil==5.2.2
+scan-build==2.0.10
```

Comment:
I would not add `scan-build` to the base requirement. It is required only for OSX now. A separate requirements file would be better.
Which will be added to the OSX install descriptions.

Reply:
It is okay for me but I am curious why don't we use 'scan-build' instead of 'LD_Logger'(if I remember correctly to the name) on Linux?